oe_theme.theme

```diff
@@ -1357,3 +1357,19 @@ function _oe_theme_trim_textfield($text, int $size = 150) {
   // Return the unchanged $text since we don't support other scenarios.
   return $text;
 }
+
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function oe_theme_preprocess_paragraph__oe_timeline(array &$variables): void {
+  $paragraph = $variables['paragraph'];
+  $variables['limit'] = $paragraph->get('field_oe_timeline_expand')->value;
+  $items = $paragraph->get('field_oe_timeline')->getValue();
+  foreach ($items as $key => $item) {
```

Comment:
There doesn't seem to be any transformation going on here, why not simply do $variables['items'] = $paragraph->get('field_oe_timeline')->getValue(); ?